### PR TITLE
fix(ci): bump cwag images to focal and adapt package setup

### LINKED ISSUE #12898: Analyse and Fix CWF integ test/cwf-integ-test/Run the integ test
see #12893

* cwag ansible used before `"deb https://facebookconnectivity.jfrog.io/artifactory/list/cwf-prod/ bionic main"` - this is not existing in magmacore
* also the vagrant configured base image is still bionic
* current approach: use focal base
  * package that does not exists in focal: python-twisted-conch - removed
  * package missing in focal: net-tools - added
  * package missing in magma repo: python-openvswitch_2.12.0-1_all.deb - added to artifactory
  * for cwag_test also missing in magma repo: libipfix-dev - added from fb repo

### PR DESCRIPTION
**Pull request by nstng**
Signed-off-by: Nils Semmelrock <nils.semmelrock@tngtech.com>

## Summary

see #12898 - (partly in pairing with @jheidbrink)

## Test Plan

* cwf/gateway$ vagrant up cwag
* cwf/gateway$ vagrant up cwag_test

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by nstng** [comment]
@Tim FYI approach for fixing the CWF integ tests

File: cwf/gateway/deploy/roles/ovs/tasks/debian.yml [hunk 1]
```diff
@@ -28,10 +28,10 @@
       - dh-autoreconf
       - libssl-dev
       - libtool
+      - net-tools
       - openssl
       - procps
       - python-all
-      - python-twisted-conch
```
Comment (jheidbrink) [inline cwf/gateway/deploy/roles/ovs/tasks/debian.yml:34]: Did the cwf integ tests succeed with this change?

File: cwf/gateway/deploy/roles/ovs/tasks/debian.yml [hunk 2]
```diff
@@ -28,10 +28,10 @@
       - dh-autoreconf
       - libssl-dev
       - libtool
+      - net-tools
```
Comment (jheidbrink) [inline cwf/gateway/deploy/roles/ovs/tasks/debian.yml:31]: just curious: how did you discover this dependency?

**Review by jheidbrink — APPROVED** [review]

File: cwf/gateway/deploy/roles/ovs/tasks/debian.yml (hunk 1 above)
Comment (nstng) [reply cwf/gateway/deploy/roles/ovs/tasks/debian.yml:34]: for now I just created the images - its on my list to try them locally (if feasible - if not, then CI might be the better choice here)

File: cwf/gateway/deploy/roles/ovs/tasks/debian.yml (hunk 2 above)
Comment (nstng) [reply cwf/gateway/deploy/roles/ovs/tasks/debian.yml:31]: net-tools contains ifconfig - ifconfig is used in one of the ansible steps

**Comment by nstng** [comment]
I did run the tests locally and get
`[vagrant@127.0.0.1:2201] out: DONE 4 runs, 44 tests, 11 skipped, 5 failures in 746.922s`
I am wondering, if I take a look at **green** runs before the magmacore artifactory switch I see results like
`[vagrant@127.0.0.1:2201] out: DONE 4 runs, 43 tests, 11 skipped, 3 failures in 1165.599s`
e.g., https://github.com/magma/magma/runs/6494088763?check_suite_focus=true. What is the goal of these tests? Are the results evaluated somewhere else?

**Review by tmdzk — APPROVED** [review]

**Comment by uri200** [comment]
It looks like ` gy_enforcement_test.go` test seems to be flaky

**Review by uri200 — APPROVED** [review]

**Pull request merged**